### PR TITLE
fix: Add min/max for some adapter options

### PR DESCRIPTION
**Pull request by Koenkk**

### CODE REVIEW

File: lib/util/settings.schema.json [hunk 1]
```diff
@@ -643,12 +643,16 @@
                     "title": "Adapter concurrency",
                     "requiresRestart": true,
                     "type": ["number", "null"],
+                    "minimum": 1,
+                    "maximum": 50,
```
Comment (Nerivec) [inline lib/util/settings.schema.json:647]: 64? for a good multiple 😉

File: lib/util/settings.schema.json [hunk 2]
```diff
@@ -643,12 +643,16 @@
...
                     "description": "Adapter concurrency (e.g. 2 for CC2531 or 16 for CC26X2R1) (default: null, uses recommended value)"
                 },
                 "adapter_delay": {
                     "type": ["number", "null"],
                     "requiresRestart": true,
                     "title": "Adapter delay",
+                    "minimum": 0,
+                    "maximum": 10000,
```
Comment (Nerivec) [inline lib/util/settings.schema.json:655]: Can probably remove a zero on the max.
Only used by deconz, seems to be 0-60 range pretty much. Have seen suggestions in various issues to go up to 200. So 1k should do.

---

Not sure what all the logic is about though? https://github.com/Koenkk/zigbee-herdsman/blob/9bded2b22da1cd84846a1b212495ea8b1021199a/src/adapter/deconz/driver/driver.ts#L697 It's not a promise.

File: lib/util/settings.schema.json [hunk 3]
```diff
@@ -711,6 +715,8 @@
                     "type": ["number", "null"],
                     "title": "Transmit power",
                     "requiresRestart": true,
+                    "minimum": -128,
+                    "maximum": 128,
```
Comment (Nerivec) [inline lib/util/settings.schema.json:719]: Is this inclusive or exclusive?
int8 range should be `>= -128 and <= 127` (node buffer).

File: lib/util/settings.schema.json (hunk 1 above)
Comment (Koenkk) [reply lib/util/settings.schema.json:647]: Done!

File: lib/util/settings.schema.json (hunk 2 above)
Comment (Koenkk) [reply lib/util/settings.schema.json:655]: Done, the logic is a bit hard to follow, but it also sets the `READY_TO_SEND_TIMEOUT`, I assume it works :)

File: lib/util/settings.schema.json (hunk 3 above)
Comment (Koenkk) [reply lib/util/settings.schema.json:719]: It's inclusive, updated it
